docs/overview/supernode_concept.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: i18n_page
+title: pages.introduction
+parent: pages.understanding_firefly
+nav_order: 1
+---
+
+# Introduction to Hyperledger FireFly
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## Your Gateway to Web3 Technologies
+
+![Hyperledger FireFly features](../images/firefly_intro_overview.png)
+
+Hyperledger FireFly is an organization's gateway to Web3, including all the blockchain ecosystems that they participate in.
+
+Multiple blockchains, multiple token economies, and multiple business networks.
+
+FireFly is not another blockchain implementation, rather it is a pluggable API Orchestration and Data layer,
+integrating into all of the different types of decentralized technologies that exist in Web3:
+
+- Public Blockchains, Layer 2 scaling solutions, Side chains & App chains
+- Permissioned Blockchains and Distributed Ledger Technologies (DLTs)
+- Decentralized storage solutions
+- Token ecosystems and standards
+- Smart Contracts, DeFi solutions and DAOs
+- Private off-chain encrypted communication rails
+- Advanced cryptography solutions
+- Identity frameworks
+
+## An Open Source Supernode for Web3 Apps
+
+Hyperledger FireFly is a toolkit for building and connecting new full-stack decentralized applications (dapps),
+as well as integrating your existing core systems to the world of Web3.
+
+It has a runtime engine, and it provides a data layer that synchronizes state from the blockchain and other Web3 technologies.
+It exposes an API and Event Bus to your business logic, that is reliable, developer friendly and ready for enterprise use.
+
+We call this a _Supernode_ - it sits between the application and the underlying infrastructure nodes,
```

Comment:
is it `SuperNode` or `Supernode`. I see the text usually refers to it as `Supernode`, but the picture below has a capitalized `N`
![Screen Shot 2022-09-15 at 11 05 22 AM](https://user-images.githubusercontent.com/10987380/190439398-ed2b480a-f78f-4555-86c5-730364d6774c.png)